ld: library not found for -lLLVM

trying to install against HEAD
brew install --HEAD julia

ld: library not found for -lLLVM
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [/private/tmp/julia-20170202-64866-18afq0f/usr/lib/libjulia.0.6.0.dylib] Error 1
make: *** [julia-src-release] Error 2



gcc -v

COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/local/Cellar/gcc/6.3.0_1/libexec/gcc/x86_64-apple-darwin16.3.0/6.3.0/lto-wrapper
Target: x86_64-apple-darwin16.3.0
Configured with: ../configure --build=x86_64-apple-darwin16.3.0 --prefix=/usr/local/Cellar/gcc/6.3.0_1 --libdir=/usr/local/Cellar/gcc/6.3.0_1/lib/gcc/6 --enable-languages=c,c++,objc,obj-c++,fortran --program-suffix=-6 --with-gmp=/usr/local/opt/gmp --with-mpfr=/usr/local/opt/mpfr --with-mpc=/usr/local/opt/libmpc --with-isl=/usr/local/opt/isl --with-system-zlib --enable-libstdcxx-time=yes --enable-stage1-checking --enable-checking=release --enable-lto --with-build-config=bootstrap-debug --disable-werror --with-pkgversion='Homebrew GCC 6.3.0_1' --with-bugurl=https://github.com/Homebrew/homebrew-core/issues --enable-plugin --disable-nls --enable-multilib
Thread model: posix
gcc version 6.3.0 (Homebrew GCC 6.3.0_1)

anything I should be trying 
tks

--- a/julia.rb
+++ b/julia.rb
@@ -62,9 +62,6 @@ class Julia < Formula
 
     # This patch ensures that suitesparse libraries are installed
     patch_list << "https://gist.githubusercontent.com/timxzl/c6f474fa387382267723/raw/2ecb0270d83f0a167358ff2a396cd6004e1b02a0/Makefile.diff"
-
-    # This patch fixes hardcoded paths to deps in deps/Makefile
-    patch_list << "https://gist.githubusercontent.com/staticfloat/cfad1fe4f69e88ec5731e5f3fd91b946/raw/22a164a7cf7c7cf2a64a089d5bfa47e25292deb2/suitesparse.mk.diff"
     return patch_list
   end
 
@@ -84,7 +81,11 @@ class Julia < Formula
 
     # Tell julia about our llvm-config, since it's been named nonstandardly
     build_opts << "LLVM_CONFIG=llvm-config-3.7"
+    build_opts << "LLVM_VER=3.7.1"
     ENV["CPPFLAGS"] += " -DUSE_ORCJIT "
+
+    # Tell julia where the default software base is, mostly for suitesparse
+    build_opts << "LOCALBASE=#{prefix}"
 
     # Make sure we have space to muck around with RPATHS
     ENV['LDFLAGS'] += " -headerpad_max_install_names"

